.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,40 @@
+name: "\U0001F680 New feature proposal"
+description: Propose a new feature to be added to PagesJS
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for your interest in the project and taking the time to fill out this feature report!
+  - type: textarea
+    id: feature-description
+    attributes:
+      label: Description
+      description: "Clear and concise description of the problem. Please make the reason and usecases as detailed as possible. If you intend to submit a PR for this issue, tell us in the description. Thanks!"
```

Comment:
I think "request" (instead of "issue") would be better, but up to you!

Reply:
It's a Github "issue" which is why I have that term.